utils/ffmpeg/install-ffmpeg-v6.0.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+rm -rf FFmpeg-n6.0 ffmpeg.zip
+echo $(pwd)
+
+curl -sL https://github.com/FFmpeg/FFmpeg/archive/refs/tags/n6.0.zip -o ffmpeg.zip
+
+unzip ffmpeg.zip
+
+cd FFmpeg-n6.0
+mkdir -p output
+./configure --prefix=$(pwd)/output --enable-gpl --enable-nonfree --enable-shared --disable-static
+
+cd ..
+echo $(pwd)
+
+#Setting PKG_CONFIG_PATH to be discoverd
+PKG_CONFIG_PATH=$(pwd)/WasmEdge/FFmpeg-n6.0/output/lib/pkgconfig cmake -Bbuild_pr2885 -GNinja -DCMAKE_BUILD_TYPE=Release -DWASMEDGE_BUILD_TESTS=ON  -DWASMEDGE_PLUGIN_FFMPEG=ON .
```

Comment:
You should not put this line here. This script should install the ffmpeg only.

Reply:
The step below the workflow will disable the AOT-related part since there is no need to build it when trying to build the plugins.
Ref: https://github.com/WasmEdge/WasmEdge/pull/2885/files#diff-6322c61775e00a19f0b58dc8e98599df3c6149301a40081bc654228ac306d44fL194

Reply:
Don't we want to compile ffmpeg? By running `make && make install` in the script file, we build ffmpeg in corresponding folder. On compilation we get this `$(pwd)/WasmEdge/FFmpeg-n6.0/output/lib/pkgconfig` path. Else the folder isn't present.

> You should not put this line here. This script should install the ffmpeg only.

Do I need to add this line in github workflow to detect ffmpeg path?

Reply:
The `make install` already builds the ffmpeg.
This `PKG_CONFIG_PATH=$(pwd)/WasmEdge/FFmpeg-n6.0/output/lib/pkgconfig ` is for building the ffmpeg plugin. You should add it to the workflow instead of the install script.

Reply:
I've updated the workflow as stated. FFmpeg compiles and build using bash script. However, the FFmpeg Plugin fails to build. It isn't detecting the FFmpeg path. I added it in the workflow.